src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowContext.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.dc;
+
+import com.powsybl.openloadflow.dc.equations.DcEquationSystem;
+import com.powsybl.openloadflow.dc.equations.DcEquationType;
+import com.powsybl.openloadflow.dc.equations.DcVariableType;
+import com.powsybl.openloadflow.equations.EquationSystem;
+// import com.powsybl.openloadflow.equations.EquationVector;
+import com.powsybl.openloadflow.equations.JacobianMatrix;
+// import com.powsybl.openloadflow.equations.TargetVector;
+import com.powsybl.openloadflow.network.LfNetwork;
+
+import java.util.Objects;
+
+/**
+ * @author Jean-Luc Bouchot (Artelys) <jlbouchot at gmail.com>
+ */
+public class DcLoadFlowContext implements AutoCloseable {
```

Comment:
Maybe we should add also an `AbstractLoadFlowContext` to share `LfNetwork` and `JacobianMatrix`

Reply:
I agree with this and we actually thought about it. This raises a question though in terms of files within the project. Today, DcLoadFlowContext is found within the OLF.dc package while the AcLoadFlowContext is found within the OLF.ac.outerloop package. Should we have an extra "abstractflow" package which would be implemented within their respective AC/DC packages?